File: Maths For IT/pyramid.cpp
#include <iostream>

using namespace std;

void create_pyramid(int height) {
    for (int i = 1; i <= height; i++) {

        for (int j = 0; j < height - i; j++) {
            cout << " ";
        }
        
        for (int j = 0; j < i; j++) {
            cout << "#";
        }
        
        cout << endl;
    }
}

int main() {
    int height;

    cout << "Enter Height: ";
    cin >> height;

    while (height < 1 || height > 8) {
        cout << "Height must be between 1 - 8: ";
        cin >> height;
    }

    create_pyramid(height);
    return 0;
}

File: Maths For IT/Pyramid Assignment/pyramid.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function create_pyramid(height) {
    for (let i = 1; i <= height; i++) {
        let row = ""

        for (let j = 0; j < height - i; j++) {
            row += " ";
        }

        for (let j = 0; j < i; j++) {
            row += "*";
        }

        console.log(row);
    }
}

function prompt_quetion() {
    rl.question('Enter Height: ', (x_prompt) => {
        let x = parseInt(x_prompt);
    
        try {
            if (x >= 1 && x <= 8) {
                create_pyramid(x);
                rl.close();
            } else {
                throw "Invalid IDK";
            }
        } catch (e) {
            prompt_quetion();
        }
    });
}

prompt_quetion();